passive/SQL injection detection.js

```diff
@@ -1,383 +1,89 @@
 // Made by kurobeats@yahoo.co.jp, regex shamelessly ripped from SQLMap project errors
 
-function scan(ps, msg, src) {
-  var url = msg.getRequestHeader().getURI().toString();
-  var body = msg.getResponseBody().toString();
-  var alertRisk = [0, 1, 2, 3, 4]; //1=informational, 2=low, 3=medium, 4=high
-  var alertConfidence = [0, 1, 2, 3, 4]; //0=fp,1=low,2=medium,3=high,4=confirmed
-  var alertTitle = [
-    "MySQL error Disclosed (script)",
-    "Postgresql error Disclosed (script)",
-    "MSSQL error Disclosed (script)",
-    "Microsoft Access error Disclosed (script)",
-    "Oracle error Disclosed (script)",
-    "IBM DB2 error Disclosed (script)",
-    "Informix error Disclosed (script)",
-    "Firebird error Disclosed (script)",
-    "SQLite error Disclosed (script)",
-    "SAP DB error Disclosed (script)",
-    "Sybase error Disclosed (script)",
-    "Ingress error Disclosed (script)",
-    "Frontbase error Disclosed (script)",
-    "HSQLDB error Disclosed (script)",
-    "",
-  ];
-  var alertDesc = [
-    "A MySQL error was discovered.",
-    "A Postgresql error was discovered.",
-    "A MSSQL error was discovered.",
-    "A Microsoft Access error was discovered.",
-    "An Oracle error was discovered.",
-    "An IBM DB2 error was discovered.",
-    "An Informix error was discovered.",
-    "A Firebird error was discovered.",
-    "An SQLite error was discovered",
-    "A SAP DB error was discovered",
-    "A Sybase error was discovered",
-    "An Ingress error was discovered",
-    "A Frontbase error was discovered",
-    "A HSQLDB error was discovered",
-    "",
-  ];
-  var alertSolution = [
-    "Ensure proper sanitisation is done on the server side, or don't. I don't care.",
-    "",
-  ];
-  var cweId = [0, 1];
-  var wascId = [0, 1];
+const ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
+
+function getMetadata() {
+  return ScanRuleMetadata.fromYaml(`
+id: 100020
+name: Information Disclosure - SQL Error
+description: An SQL error was found in the HTTP response body.
+solution: >
+  Ensure proper sanitisation is done on the server side.
+risk: high
+confidence: medium
+cweId: 209  # CWE-209: Generation of Error Message Containing Sensitive Information
+wascId: 13  # WASC-13: Information Leakage
+status: alpha
+codeLink: https://github.com/zaproxy/community-scripts/blob/main/passive/Find%20Internal%20IPs.js
```

Comment:
Needs update. (We could add a test that checks the URL matches the path/filename, not asking for this PR.)